eo_protocol.txt

```diff
@@ -2799,25 +2859,25 @@ client_packet(Spell, Target_Other)
 }
 
 "Group spell cast"
-client_packet(Spell, Target_Group)
+client_packet(Magic, Target_Group)
 {
 	short spell_id
 	three timestamp
 }
 
 "Nearby player chanting a spell"
-server_packet(Spell, Request)
+server_packet(Magic, Request)
 {
 	short player_id
 	short spell_id
 }
 
 "Nearby player self-casted a spell"
-server_packet(Spell, Target_Self)
+server_packet(Magic, Target_Self)
 {
 	short player_id
 	short spell_id
-	int spell_heal_hp
+	int Magic_heal_hp
```

Comment:
I've been skimming pretty quickly, but you might want to double-check that your find/replace didn't mess up the casing anywhere else

Reply:
Ah good catch, will scan for others. I did the first few replaces recklessly and then one at a time after 😓